devtools-har-extension/contentscript.js

```diff
@@ -0,0 +1,31 @@
+/* global chrome */
+(function (window, chrome) {
+  window.__sendPageTimings = function () {
+    var page = {
+      'pageTimings': {
+        'onContentLoad': window.performance.timing.domContentLoadedEventEnd - window.performance.timing.fetchStart,
+        'onLoad': window.performance.timing.loadEventEnd - window.performance.timing.fetchStart
+      },
+      'startedDateTime': new Date(window.performance.timing.fetchStart).toISOString(),
+      'title': window.location.href
+    }
```

Comment:
Even though the extension is mostly concerned about the HAR and network requests, sending the performance timings is a nice small addition. It would be better to pass the actual window.performance object which is serialisable and let the client / consumer worry about how to consume it.
```
let perfData = {
  performance: window.performance,
  url: window.location.href
}

chrome.runtime.sendMessage({
  source: 'devtools-har-extension',
  type: 'page-performance',
  data: perfData
}) 
```

Can you also include more information about how this info can be consumed?